aip/general/0009.md

```diff
@@ -102,6 +102,22 @@ The version of an API or a group of APIs if they are defined
 together. An API Version is often represented by a string, such as "v1", and
 presents in API requests and Protocol Buffers package names.
 
+### Client
+
+Code that calls the API or operates on the resource data at rest. These clients
+may be a program that performs a specific task on an API, or be a generic tool
+that exposes the API in a more user-accessible fashion (e.g. a command line
+interface).
+
+Examples of clients include:
```

Comment:
Again, subject verb agreement is missing. Also, colons require us of some form of "following", e.g. "as follows" or "includes the following". So, you can do,

> Examples of clients include the following:

or leave it as is and `s/:/,/`.

For each bullet you need to make the subject of the phrase plural to fix the subject verb agreemnent, e.g. "Command line interfaces". I would remove the periods from each bullet, because they are not sentences.

Reply:
ack. made everything plural, went with your suffix of "the following". PTAL.